www/content/tutorials/getting-started.md

```diff
@@ -7,3 +7,15 @@ weight = 1
 +++
 
 {{< wip >}}
+# Build from Source
+
+Clone from repository
+```shell
+git clone git@github.com:linkdd/tricorder.git
+cd tricorder
+```
+Compile from source (requirement rust installed via rustup):
```

Comment:
Maybe list the requirements before any commands, and specify a minimum version supported?

Reply:
I added a table in the next commit, but I don't know the minimal version. Can you help me there?

Reply:
By requirements I meant "rust installed via rustup". It's actually just rust and cargo, installed via any means.

On my computer I have Rust/Cargo 1.65.

Reply:
Ok :)